content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -506,13 +529,13 @@ EOF
 Retry the request without a token. The request now fails with error code `403`:
 
 {{< text bash >}}
-$ curl $INGRESS_HOST/headers -s -o /dev/null -w "%{http_code}\n"
+$ curl "$INGRESS_HOST:$INGRESS_PORT"/headers -s -o /dev/null -w "%{http_code}\n"
 403
 {{< /text >}}
 
 ### Require valid tokens per-path
 
-To refine authorization with a token requirement per host, path, or method, change the authorization policy to only require JWT on `/headers`. When this authorization rule takes effect, requests to `$INGRESS_HOST/headers` fail with the error code `403`. Requests to all other paths succeed, for example `$INGRESS_HOST/ip`.
+To refine authorization with a token requirement per host, path, or method, change the authorization policy to only require JWT on `/headers`. When this authorization rule takes effect, requests to `"$INGRESS_HOST:$INGRESS_PORT"/headers` fail with the error code `403`. Requests to all other paths succeed, for example `"$INGRESS_HOST:$INGRESS_PORT"/ip`.
```

Comment:
```suggestion
To refine authorization with a token requirement per host, path, or method, change the authorization policy to only require JWT on `/headers`. When this authorization rule takes effect, requests to `$INGRESS_HOST:$INGRESS_PORT/headers` fail with the error code `403`. Requests to all other paths succeed, for example `$INGRESS_HOST:$INGRESS_PORT/ip`.
```

Reply:
Good catch.  I moved through the search and replace too fast.  Should be fixed in the new version.